src/blocks/cards.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  // height: 406px;
+
+  width: 200px;
+  background-color: $four-color;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__top {
+    padding-left: 3px;
+    padding-right: 3px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    background-repeat: no-repeat;
+    background-size: auto;
+  }
+
+  &__description {
+    width: 166px;
+    height: 36px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $triple-color;
+
+    &--margin--top {
+      margin-top: 40px;
+    }
+  }
+
+  &__code {
+    width: 98px;
+    height: 14px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
```

Comment:
Ensure that the variable `$secondary-color` is defined in your SCSS files. If it's not defined, it will cause a compilation error.